client/java/src/main/java/com/linecorp/centraldogma/internal/client/AbstractWatcher.java

```diff
@@ -258,6 +262,12 @@ private void doWatch(int numAttemptsSoFar) {
                          logger.info("{}/{}{} does not exist yet; trying again",
```

Comment:
I think we shouldn't log this when `errorOnEntryNotFound` is `true` because we don't try again. 😉